File: 2144-minimum-cost-of-buying-candies-with-discount/2144-minimum-cost-of-buying-candies-with-discount.py
class Solution(object):
    def minimumCost(self, cost):
        cost.sort()
        n=len(cost)
        ans=0
        took=0
        for i in range(n-1,-1,-1):
            if took==2:
                took=0
            else:
                ans+=cost[i]
                took+=1
        return ans

File: 2144-minimum-cost-of-buying-candies-with-discount/2144-minimum-cost-of-buying-candies-with-discount.java
class Solution {
    public int minimumCost(int[] cost) {
        Arrays.sort(cost);
        int n = cost.length;
        int temp = 0;
        int count = 0;
        for (int i = n - 1; i >= 0; i--) {
            if (count == 2) {
                count = 0;
            } else {
                temp += cost[i];
                count++;
            }

        }
        return temp;
    }
}